src/SignOut.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import Container from 'react-bootstrap/lib/Container';
+
+import auth from './auth';
+
+class SignOut extends Component {
+
+  componentDidMount() {
+    auth.removeFromHeader();
```

Comment:
Actually, might be better to do this immediately after success of the sign-out request.

Reply:
are you referring to `sign-out` on the play app?

Reply:
No I mean wherever in the client app we actually make the request to the `/sign-out` endpoint--like probably from a nav link or something--in the `.then` callback on success.

Reply:
oh got it. i can try to add navbar